### PR TITLE
Prepare 2.8 stable release

### PR DESCRIPTION
**Pull request by eguzki**

### CODE REVIEW

File: pkg/3scale/amp/manual-templates/amp/apicast.yml [hunk 1]
```diff
@@ -121,7 +121,7 @@ parameters:
   value: "2.7.0"
```
Comment (miguelsorianod) [inline pkg/3scale/amp/manual-templates/amp/apicast.yml]: The AMP_RELEASE value should be updated for apicast template too

**Comment by eguzki** [comment]
ready for review @miguelsorianod

**Comment by eguzki** [comment]
@miguelsorianod new commits added. Review please

**Comment by miguelsorianod** [comment]
Looks good to me but a bug in system image has been detected.
This will require updating the porta image (and its quay url). We can merge it now and then update it in another PR when the new image is available and tested by qe or we can wait and update it in this PR.

**Pull request merged**